### ISSUE TITLE
stripH=off option still removes hydrogens 

### ISSUE THREAD

**Issue by mmagithub** [gnina/gnina#163]
### Issue summary
Hi,
I am wondering what is the proper way of preventing Hydrogen removals from input sdf files in the outputs. 
It seems that the 'stripH=off' option still removes hydrogen atoms provided within the sdf files, whereas I still want to keep these hydrogens in the outputs,

It does not matter providing the keywords in the command line or the configuration file.

Any suggestions ?

Thanks

### Your system configuration
Operating system:
Compiler:
CUDA version (if applicable):
CUDNN version (if applicable):
Python version:

**Comment by recursingfeynman** [comment]
Be sure to use boolean arguments in this option, according to the [source code implementation](https://github.com/gnina/gnina/blob/7af58489c6086a0d5c4481bad12a9c9bfdbc90d2/gninasrc/main/main.cpp#L1243).

**Comment by nicoladmSB** [comment]
> Be sure to use boolean arguments in this option, according to the [source code implementation](https://github.com/gnina/gnina/blob/7af58489c6086a0d5c4481bad12a9c9bfdbc90d2/gninasrc/main/main.cpp#L1243).

Correct, but if the invocation is bad, `gnina` won't run, so I believe @mmagithub ran it correctly. In the context of `boost::program_options`, [many values](https://www.boost.org/doc/libs/1_85_0/doc/html/program_options/overview.html#Types) count as true/false:

> When a boolean does take a parameter, there are several options. Those that evaluate to true in C++ are: "true", "yes", "on", "1". Those that evaluate to false in C++ are: "false", "no", "off", "0".

----

At any rate, I can confirm the issue: `stripH` has no effect on whether or not hydrogens are present in the output file.

It would be a useful feature to retain or re-add hydrogens to outputs, even respecting the initial atom ordering in the ligand file.

**Comment by dkoes** [comment]
Nonpolar hydrogens are not kept around. `stripH` does have an effect, but only for polar hydrogens.
